src/webcomponents/helperfunctions/jsonToCol.ts

```diff
@@ -0,0 +1,40 @@
+import { JsonToCol, LanguagesInput } from "../../@types/interfaces";
+
+const mapLangToSheetColumns = (languages: LanguagesInput) => {
+  let columnTitle: Array<object> = Object.keys(
+    languages
+  ).map((title: string) => ({ type: "text", title: `${title}`, width: 150 }));
```

Comment:
this should be simplified like this `).map((title) => ({ type: "text", title, width: 150 }));`